README.md

```diff
@@ -10,3 +10,30 @@ project.
 
 ## Who should use this project?
 
+You should use this project if you're looking for a quick way to implement an
+Open Service Broker.
+
+## Getting started
+
+You can just clone this project and deploy it to start poking around right away:
+
+```console
+# clone the repo
+$ git clone git://github.com/pmorie/go-open-service-broker-client
+
+# deploy with helm
```

Comment:
Considering we are mentioning Helm here, would we need service-catalog as well then?